src/test/kotlin/no/nav/helsearbeidsgiver/domene/inntektsmelding/v1/skjema/SkjemaInntektsmeldingTest.kt

```diff
@@ -100,6 +136,26 @@ class SkjemaInntektsmeldingTest :
 
                     skjema.valider() shouldBe setOf(Feilmelding.AGP_MAKS_16)
                 }
+                context("behandlingsdager") {
+
+                    fun List<Periode>.tilArbeidsgiverperiode() = Arbeidsgiverperiode(this, emptyList(), null)
+                    val behandlingsdager = List(12) { Periode(1.januar.plusWeeks(it.toLong()), 1.januar.plusWeeks(it.toLong())) }
+
+                    test("kan være 12 dager med en uke mellomrom") {
+                        behandlingsdager.tilArbeidsgiverperiode().valider().shouldBeEmpty()
+                    }
+
+                    test("kan være 12 dager over årsskifte") {
```

Comment:
Denne er fsv grei siden vi ikke skal validere den faktiske regelen "en i uka, hver uke" - men kanskje vi ikke trenger å sjekke på dette i det hele tatt egentlig... Evt heller sette opp en test som går hver uke fra desember - det jeg lurer på, er om man kan "snike" inn to behandlingsdager i samme uke, før og etter nyttårsaften! Mistenker at det kanskje er mulig nå, og det bør ikke gå an